server/api/collections.test.js

```diff
@@ -198,6 +198,78 @@ describe("#collections.move", () => {
     expect(updateCreatedCollection.data.index).toEqual("h");
     expect(updateCreatedCollection.data.index > "Q").toBeTruthy();
   });
+
+  it("if index collision with an extra collection, should updated index of other collection", async () => {
```

Comment:
Is it possible to tweak the remove collision method so that the collection that gets the updated `index` is the one being acted upon, rather than the one it happens to be "next" to? 

I think that'll make a lot more sense. The `index` you provide to create/update is a clue as to where the collection should land visually rather than a need for it to be that specific character.

Reply:
I made it work this way because when I send an index through API as a developer I expect the collection's index to be that specific value. I don't care about the index of other collections. So I changed the index of other collections giving precedence to the one being acted upon.

Reply:
Tom, What do you think? I can always rewrite the function to behave the way you want.

Reply:
Personally I think the specific value is not important, the thing that's important is the position in the list but I'm very conscious of the amount of iteration we've already asked you to put into the PR.

One thing I wonder that might tip your thinking is that if we went the route of modifying the incoming `index` does that simplify the event data format too also as you wouldn't need to store an array of tuples? As only one collection index would ever be changed…

Reply:
Indeed, this will definitely simplify things. I will jump right into it.

Reply:
Done 🎉

Reply:
so CLEAN

Reply:
Ikr! Thanks for the nudge. I don't mind iterating a hundred times if the code quality improves.